build.gradle

```diff
@@ -883,6 +884,36 @@ gradle.taskGraph.whenReady({ graph ->
   tasks.getByName('allReports').reportOn rootProject.subprojects.collect{ it.tasks.withType(Test) }.flatten()
 })
 
+task installPulse {
+  def pulseWarName = "pulse-${pulseVersion}.war"
+  def pulseDir = System.env.pulsedir
+
+  if (pulseDir == null || pulseDir.length() == 0) {
+    pulseDir = "${projectDir}/../pulse"
+  }
+
+  if (pulseDir == null || pulseDir.length() == 0) {
+    throw  new Exception(" Pulse directroy can not be empty while building product")
+  }
+
+  println "Using Pulse location ${pulseDir}"
+
+  println runScript("${pulseDir}/build.sh", pulseDir, ["clean", "build-all"]);
+
+  doFirst {
+    delete "${snappyProductDir}/lib/pulse.war"
+  }
+
+  doLast {
+    copy {
+      from "${pulseDir}/build-artifacts/linux/dist/${pulseWarName}"
+      into "${snappyProductDir}/lib/"
+      include "${pulseWarName}"
+      rename { filename -> "pulse.war" }
```

Comment:
Any specific reason for the rename? Does the product depend on the name being exactly pulse.war?